docs/en/routing/introduction.md

```diff
@@ -108,7 +108,7 @@ Query parameters can also be added to route URLs. As with normal query parameter
 ```ts
 export default [
 	{
-		path: 'home/{page}',
+		path: 'home/{page}?queryOne=Friends&queryTwo=Family',
```

Comment:
I don't think this is correct for routing, it should be `path: 'home/{page}?{queryOne}&{queryTwo}` and if there are default values then they can be defined in the `defaultParams` section of the config.